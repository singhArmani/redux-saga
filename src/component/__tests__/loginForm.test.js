import React from 'react';
import renderer from 'react-test-renderer';
import LoginForm from '../loginForm';

describe('LoginForm', () => {
  const testProps = {
    signIn: jest.fn(() => ({})),
    isUserSigningIn: false
  };
  it('renders correctly', () => {
    const tree = renderer.create(<LoginForm {...testProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
