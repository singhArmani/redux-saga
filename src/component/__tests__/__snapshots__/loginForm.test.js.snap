// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm renders correctly 1`] = `
<form>
  <div
    className="form-group"
  >
    <label
      className="control-label"
      htmlFor="formHorizontalEmail"
    >
      Username
    </label>
    <input
      className="form-control"
      id="formHorizontalEmail"
      name="username"
      onChange={[Function]}
      placeholder="Your username"
      type="text"
      value=""
    />
  </div>
  <div
    className="form-group"
  >
    <label
      className="control-label"
      htmlFor="formHorizontalPassword"
    >
      Password
    </label>
    <input
      className="form-control"
      id="formHorizontalPassword"
      name="password"
      onChange={[Function]}
      placeholder="Your password"
      type="password"
      value=""
    />
  </div>
  <button
    className="btn btn-primary"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    Sign In
  </button>
</form>
`;
